docs/docs/guides/core-types/evaluation_playground.md

```diff
@@ -0,0 +1,109 @@
+
+### Add a dataset
+
+[Datasets](../core-types/datasets) are organized collections example user inputs and your expected responses from models. During evaluation, the playground feeds each test input into your model, collects its output, and then scores the output based on a metric you've selected, such as correctness. You can choose to create a dataset in the UI, add an existing dataset already saved to your project, or upload a new dataset.
+
+You can upload datasets in the following formats:
+
+* `.csv`
+* `.tsv` 
+* `.json`
+* `.jsonl`
+
+See the Datasets pages for more information on how to format and save datasets to Weave.
+
+To add a dataset in the **Dataset** section: 
+
+1. Click the dropdown menu and then select either: 
+    * **Start from scratch** to create a new dataset in the UI.
+    * **Upload a file** to upload a dataset from your local machine.
+    * An existing dataset already saved to your project.
+2. Optional: Click **Save** to save the dataset to your project for later use.
+
+Once you've selected an option, the dataset is visible in the right pane of the UI and you can edit each field as necessary by clicking on it. You can also add new rows to the dataset by clicking **Add row**.
+
+### Add a model
+
+[Models](https://docs.wandb.ai/guides/models/) are trained systems that generate output based on user input. You can select existing models in your project to create new ones to evaluate.
+
+To add a model in the **Models** section of the Evaluation Playground:
+
+1. Click **Add Model** and either select **New Model** or an existing model from the dropdown menu.
+2. If you selected **New Model**, configure the following fields:
+    * **Name**: Add a descriptive name to your new model.
+    * **LLM Model**: Select a foundation model to build your new model on, such as OpenAI's GPT-4. You can select from a list of foundation models you have already configured access to, or you can add access to a foundation model by selecting **Add AI provider** and selecting a model. Adding a provider prompts you to enter your access credentials to that provider. See your provider's documentation to see how to locate your API key, endpoints, and any additional configuration information you need to access the model using Weave.
+    * **System Prompt**: Provide the model instructions on how it should behave, for example, `You are a helpful assistant specializing in Python programming.`
+
+    If you choose an existing model, a new field appears beside the model's name that allows you to select a version of the existing model and there are no other additional fields to configure. If you want to make changes to your existing model before or after evaluation, use the [Prompt Playground](../tools/playground.md).
+
+3. Optional: Click **Save** to save the model to your project for later use.
+4. Optional: You can add additional models to simultaneously evaluate by clicking **Add Model** again and adding other models as necessary.
+
```

Comment:
It might be critical to call out that the column with name `user_input` is used as the subsequent message after the system prompt

Reply:
I've added this in the **System Prompt** bullet.